pkg/controller/controller.go

```diff
@@ -108,16 +101,14 @@ func (c *Controller) RunInformers(stopCh <-chan struct{}) {
 		return workplans[i].CreationTimestamp.After(workplans[j].CreationTimestamp.Time)
 	})
 
+	workflowKeys := make(map[string]struct{})
```

Comment:
You could use `sets.String`. https://github.com/kubernetes/apimachinery/blob/master/pkg/util/sets/string.go